### ISSUE TITLE
Any plan to support using jsx inside the description attribute? 

### ISSUE THREAD

**Issue by alexya** [kamranahmedse/driver.js#428]
The requirements need us to show a complex page/content in the user tours, and the `description` of text and HTML is very hard for us to implement. 
I wonder if there is a plan to support using jsx inside the `description` attribute.  
or is there any workaround?

**Comment by andromidasj** [comment]
Agreed, passing in a plain string is pretty limiting, even if it allows HTML. Passing in JSX, in React for example, would be great - also allowing for better customization since we could potentially use our own custom styled components within.

**Comment by guifeliper** [comment]
Is there any solution yet?

**Comment by saikat709** [comment]
Hi @nilbuild, I came across this issue and would be interested in working on it. Supporting JSX/custom React content in the `description` (and potentially `title`) would be a really useful addition.

Would you be open to a contribution for this? If so, I’d be happy to look into the current implementation and work on a PR.
